apps/web-giddh/src/app/shared/shared.module.ts

```diff
@@ -250,7 +254,7 @@ export class PickDateAdapter extends NativeDateAdapter {
         CurrencyModule,
         PrimarySidebarComponent,
         TranslateDirectiveModule,
-        ValidateSectionPermissionDirectiveModule
```

Comment:
Please check this should not be reverted

Reply:
done